src/Runner/Runner.php

```diff
@@ -168,14 +168,9 @@ private function fixFile(\SplFileInfo $file, LintingResultInterface $lintingResu
                 }
             }
         } catch (\Exception $e) {
-            $this->dispatchEvent(
-                FixerFileProcessedEvent::NAME,
-                FixerFileProcessedEvent::create()->setStatus(FixerFileProcessedEvent::STATUS_EXCEPTION)
-            );
-
-            $this->errorsManager->report(new Error(Error::TYPE_EXCEPTION, $name));
-
-            return;
+            return $this->processException($name);
+        } catch (\Throwable $e) {
```

Comment:
this way here is by far the best way to handle this, and that's why everyone does it this way